### ISSUE TITLE
Window cannot be moved more than one workspace when popup is open

### ISSUE THREAD

**Issue by mzur** [mzur/gnome-shell-wsmatrix#220]
If I move a window to another workspace with <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>Shift</kbd>+<kbd>Arrow</kbd> and then want to move it further while the switcher popup is open, I get:

```
JS ERROR: TypeError: window is null
actionMoveWindow@resource:///org/gnome/shell/ui/windowManager.js:1848:13
_showWorkspaceSwitcher@/home/m/.local/share/gnome-shell/extensions/wsmatrix@martin.zurowietz.de/workspacePopup/workspaceManagerOverride.js:428:25
_keyPressHandler@/home/m/.local/share/gnome-shell/extensions/wsmatrix@martin.zurowietz.de/workspacePopup/workspaceSwitcherPopup.js:185:26
vfunc_key_press_event@resource:///org/gnome/shell/ui/switcherPopup.js:186:18
```

And nothing happens. The window can only be moved again when the popup was closed.

**Comment by Dylansc22** [comment]
Same issue for me!
Ubuntu 22.04.1 LTS x86_64 using 
GNOME 42.2

I noticed if you just "switch workspaces" (pressing Ctrl+Alt+Arrow), and then release the Arrow, and then press Shift (so now Ctrl+Alt+Shift are all being pressed down), if you then click your arrow direction triggering a "move window to workspace" event, you can move the window multiple workspaces with multiple Arrow presses.

Certainly not a solution, but it makes it tolerable enough while I *hope* for a fix! hah.

**Comment by pdecat** [comment]
Reducing the `Time to show the popup (ms)` parameter to something low (but not 0) like 50ms makes it tolerable for me.

**Comment by esauvisky** [comment]
Fixed by #246
